packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMyMessageComponent.tsx

```diff
@@ -159,7 +159,8 @@ export const ChatMyMessageComponent = (props: ChatMyMessageComponentProps): JSX.
           text,
           /* @conditional-compile-remove(file-sharing-acs) */
           {
-            attachments: attachments
+            attachments: attachments,
+            type: props.richTextEditorOptions ? 'html' : 'text'
```

Comment:
As of today, COntoso can't change message type, do we need to provide this option then?